Ports/julius/package.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port="julius"
```

Comment:
Our preferred style is to use single quotes for all non-interpolated strings :^)